src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,42 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private final int EMPTY = -1;
 
-    public int total = -1;
+    private final int MAX_INDEX = 11;
 
-    public void countIn(int in) {
+    private final int MAX_AMOUNT_OF_ELEMENTS = 12;
+
+    private int[] numbers = new int[MAX_AMOUNT_OF_ELEMENTS];
+
+    private int total = -1;
+
+    public int getTotal() {
+        return total;
+    }
+
+    public void countIn(final int in) {
         if (!isFull())
             numbers[++total] = in;
     }
 
     public boolean callCheck() {
-        return total == -1;
+        return total == EMPTY;
     }
 
     public boolean isFull() {
-        return total == 11;
+        return total == MAX_INDEX;
     }
 
     protected int peekaboo() {
         if (callCheck())
-            return -1;
+            return EMPTY;
```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)